packages/main/src/plugin/mcp/mcp-registry.ts

```diff
@@ -549,18 +553,30 @@ export class MCPRegistry {
     // Validate the ServerList data but continue even if invalid
     this.schemaValidator.validateSchemaData(data, 'ServerList', registryURL);
 
+    // Validate each individual ServerResponse and mark invalid ones
+    const validatedServers = data.servers.map(serverResponse => {
+      const isValid = this.schemaValidator.validateSchemaData(serverResponse, 'ServerResponse', registryURL, true);
```

Comment:
isssue(blocking): I think this is causing performance issue, the interface looks laggy at start.

For example with the https://registry.modelcontextprotocol.io/,

we call validateSchemaData:
- ServerList: 151 times
- ServerResponse: 4441